.gitignore

```diff
@@ -1,2 +1,4 @@
 *.env*
 __pycache__
+.vscode/
+data_test/
```

Comment:
if you want a test data folder change it to:
```suggestion
tests/data
```

it is the most common way to do that